test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala

```diff
@@ -991,6 +991,61 @@ class CaseSpec extends BaseFeatureSpec {
 
   }
 
+  feature("Get Case sorted by reference") {
+
+    val case1 = createCase().copy(reference = "1")
+    val case2 = createCase().copy(reference = "2")
+
+    scenario("Sorting default - ascending order") {
+      Given("There are few cases in the database")
+      storeCases(case1, case2)
```

Comment:
could you store them in the opposite order?
otherwise we are not sure it works...

```
storeCases(case2, case1)
```

Reply:
I am sure :)

Reply:
actually I saw there is another scenario below, where you cover this